src/puck.ts

```diff
@@ -271,11 +287,23 @@ export class RikaiPuck {
     target.dispatchEvent(mouseEvent);
   };
 
+  private clickState: ClickState = { kind: 'idle' };
+  private static readonly doubleClickHysteresis = 300;
+
   private readonly onPuckPointerDown = (event: PointerEvent) => {
     if (!this.enabled || !this.puck) {
       return;
     }
 
+    if (this.clickState.kind === 'idle') {
+      this.clickState = { kind: 'gotpointerdown' };
```

Comment:
(If we go with the suggested states above, we'll likely want to start a timeout here to differentiate between clicks and drags.)

Reply:
Handled in 779a08a; search for:

`this.clickState = { kind: 'dragging' };`